src/app/app.component.html

```diff
@@ -1,7 +1,7 @@
-<!DOCTYPE html>
-
-<!-- Probably just a placeholder html file for now. Unsure of where to put each individual page as of right now, but we'll figure it out later. -->
+<!DOCTYPE html><!-- Probably just a placeholder html file for now. Unsure of where to put each individual page as of right now, but we'll figure it out later. -->
 
 <app-navbar></app-navbar> <!-- Static Navbar - Anthony -->
 
-<router-outlet></router-outlet> <!-- This actually allows us to determine what buttons display what pages SUPER simply, took me a second to figure out - Anthony -->
\ No newline at end of file
+<div class="content">
+    <router-outlet></router-outlet>
+</div>
```

Comment:
the footer component requires another line in this file after this line
<app-footer></app-footer>